### PR TITLE
Fixes for arm 7 including continuous build

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: cmds/ip/ip.go [hunk 1]
```diff
@@ -137,12 +137,13 @@ func showLinkAddresses(w io.Writer, link netlink.Link) {
 		fmt.Fprintf(w, " scope %s %s\n", addrScopes[netlink.Scope(addr.Scope)], addr.Label)
 
 		var validLft, preferredLft string
-		if addr.PreferedLft == math.MaxUint32 {
+		// TODO: fix vishnavanda/netlink. *Lft should be uint32, not int.
```
Comment (rjoleary) [inline cmds/ip/ip.go]: Did you open a bug with them?

**Comment by rminnich** [comment]
there is a bug open. I'll add closes.

**Comment by rminnich** [comment]
fixes #537

File: cmds/ip/ip.go (hunk 1 above)
Comment (rminnich) [reply cmds/ip/ip.go]: oh woops misunderstood. Yeah I'll do that.

**Pull request merged**